.github/workflows/ci.yml

```diff
@@ -35,6 +35,11 @@ jobs:
     - uses: cgrindel/gha_set_up_bazel@v1
       with:
         repo_name: rules_bazel_integration_test
+        # Disable local disk and repository caches related to file permission 
```

Comment:
This looks reasonable; I'd also consider running with `--nohome_rc` and/or `--nosystem_rc`, but not clear which is more/better targeted :)

Reply:
In this case, the `--disk_cache` and `--repository_cache` settings are written by the `gha_set_up_bazel` GitHub action. So, not loading the home or system bazelrc files would not help.